nni/retiarii/nn/pytorch/mutator.py

```diff
@@ -63,30 +63,64 @@ def mutate(self, model):
 
 class ValueChoiceMutator(Mutator):
     def __init__(self, nodes: List[Node], candidates: List[Any]):
+        # use nodes[0] as an example to get label
         super().__init__(label=nodes[0].operation.parameters['label'])
         self.nodes = nodes
         self.candidates = candidates
 
     def mutate(self, model):
         chosen = self.choice(self.candidates)
+        # no need to support transformation here,
+        # because it is naturally done in forward loop
         for node in self.nodes:
             target = model.get_node_by_name(node.name)
             target.update_operation('prim::Constant', {'type': type(chosen).__name__, 'value': chosen})
 
 
+class ParameterChoiceLeafMutator(Mutator):
+    # mutate the leaf node (i.e., ValueChoice) of parameter choices
+    # should be used together with ParameterChoiceMutator
+
+    def __init__(self, candidates: List[Any], label: str):
+        super().__init__(label=label)
+        self.candidates = candidates
+
+    def mutate(self, model: Model) -> Model:
+        # leave a record here
+        # real mutations will be done in ParameterChoiceMutator
+        self.choice(self.candidates)
+
+
 class ParameterChoiceMutator(Mutator):
```

Comment:
better to give more explanation for this class.

Reply:
for example, to claim that this is to deal with the ValueChoice that is used as class (basic unit) or function's input argument